playground/backend/internal/fs_tool/fs.go

```diff
@@ -117,5 +127,9 @@ func (l *LifeCycle) GetAbsoluteExecutableFilesFolderPath() string {
 
 // getFileName returns fileName by pipelineId and fileType ({pipelineId}.java for java SDK).
 func getFileName(pipelineId uuid.UUID, fileType string) string {
-	return fmt.Sprintf("%s.%s", pipelineId, fileType)
+	if fileType != "" {
+		return fmt.Sprintf("%s.%s", pipelineId, fileType)
+	} else {
```

Comment:
and in general I would use inheritance (we have goLifeCycle and javaLC) to override method and wouldn't use if-else

Reply:
got it